src/node/scripts.h

```diff
@@ -2,12 +2,16 @@
 // Licensed under the Apache 2.0 License.
 #pragma once
 
-#include "kv/map.h"
+#include "kv/experimental.h"
 #include "script.h"
 
 namespace ccf
 {
-  using Scripts = kv::Map<std::string, Script>;
+  using Scripts = kv::experimental::Map<
```

Comment:
Here's one example of using this in a real map, where we really want governance state in trivially-readable JSON rather than msgpack. In a future PR I'll do a pass like this on more governance tables, and remove the now unused msgpack serialisation machinery.